src/main/java/md/miller1995/Dealership/services/auth/AuthService.java

```diff
@@ -0,0 +1,28 @@
+package md.miller1995.Dealership.services.auth;
+
+import md.miller1995.Dealership.models.dto.auth.UserAuthDTO;
+import md.miller1995.Dealership.models.dto.auth.UserRegisterDTO;
+import md.miller1995.Dealership.utils.AuthResponse;
+
+public interface AuthService {
+
+    /**
+     * This method is used to register a new user in the database
+     *
+     * @author Anton Nirca
+     * @since  21/05/2023
+     *
+     * @param userRegisterDTO
+     */
+    void register(UserRegisterDTO userRegisterDTO);
```

Comment:
As I was saying in the controller, would this method help us if we return `UserRegisterDTO`?